Foundation/URLSession/URLSession.swift

```diff
@@ -212,8 +212,6 @@ open class URLSession : NSObject {
     fileprivate static let _shared: URLSession = {
         var configuration = URLSessionConfiguration.default
         configuration.httpCookieStorage = HTTPCookieStorage.shared
```

Comment:
I think the cookie storage line can also be removed.

Reply:
Yeah, it's redundant now. Still, one thing at a time.